pages/home/script.js

```diff
@@ -499,12 +575,12 @@ const showHazardDetails = (hazardReport) => {
       if (windowWidth.matches) {
         reportShareBtn.style.display = 'flex';
         reportCloseBtn.style.display = 'flex';
-        content.style.height = 'fit-content';
+        // content.style.height = 'fit-content';
       } else {
         reportShareBtn.style.display = 'none';
         reportCloseBtn.style.display = 'none';
         // Set initial height
-        updateHeight(40);
+        // updateHeight(40);
```

Comment:
Double-check with @ReinhardtBotha, I think this updateHeight is used by the pull-up card

Reply:
It's not required as I tested it, will discuss with him and remove the commented lines